ldt/arch/Config.pl

```diff
@@ -407,15 +417,26 @@
 if($use_hdf4 == 1) {
    if(defined($ENV{LDT_HDF4})){
       $sys_hdf4_path = $ENV{LDT_HDF4};
-      $inc = "/include/";
+      if($vsc_tier_system eq "genius"){
+         $inc = "/include/hdf/";
+      }
+      elsif($vsc_tier_system eq "hortense") {
+         $inc = "/include/";
+      }
+      else{
+         print "--------------ERROR---------------------\n";
+	 print "VSC_SYSTEM not available in Config.pl\n";
+         print "--------------ERROR---------------------\n";
+	 exit 1;
+      }
       $lib = "/lib/";
       $inc_hdf4=$sys_hdf4_path.$inc;
       $lib_hdf4=$sys_hdf4_path.$lib;
    }
    else {
       print "--------------ERROR---------------------\n";
       print "Please specify the HDF4 path using\n";
-      print "the LDT_HDF4 variable.\n";
+      print "the LIS_HDF4 variable.\n";
```

Comment:
not sure why LDT_ is explicitly changed to LIS_ in a file for ldt

Reply:
Fixed.